README.md

```diff
@@ -41,23 +42,31 @@ You can directly install the package from PyPI.
 pip install power-grid-model
 ```
 
+## Install from Conda
```

Comment:
i noticed the `Description` is still empty in https://anaconda.org/conda-forge/power-grid-model